include/wx/string.h

```diff
@@ -2560,6 +2570,10 @@ class WXDLLIMPEXP_BASE wxString
       if ( !ch.IsAscii() )
           m_impl.assign(wxStringOperations::EncodeNChars(n, ch));
       else
+#elif wxUSE_UNICODE_WCHAR && SIZEOF_WCHAR_T == 2
```

Comment:
No, it isn't, because `!ch.IsAscii()` is under `#if wxUSE_UNICODE_UTF8` block.

Maybe I should change the code from
```c++
#if wxUSE_UNICODE_UTF8
      if ( !ch.IsAscii() )
          m_impl.assign(wxStringOperations::EncodeNChars(n, ch));
      else
#endif
          m_impl.assign(n, (wxStringCharType)ch);
```
to
```c++
#if wxUSE_UNICODE_UTF8
      if ( !ch.IsAscii() )
          m_impl.assign(wxStringOperations::EncodeNChars(n, ch));
      else
          m_impl.assign(n, (wxStringCharType)ch);
#elif wxUSE_UNICODE_UTF16
      if ( !ch.IsBMP() )
          m_impl.assign(wxStringOperations::EncodeNChars(n, ch));
      else
          m_impl.assign(n, (wxStringCharType)ch);
#else
      m_impl.assign(n, (wxStringCharType)ch);
#endif
```
for readability.